cmd/keygen/handler.go

```diff
@@ -80,33 +80,29 @@ func runKeygen(verbose bool, configPath, keyPath string, force bool, keygenPath
 			return fmt.Errorf("keys already exist at %s", finalKeyPath)
 		}
 	}
-	
+
 	jwtManager := jwt.NewManager(logger)
-	
+
 	if err := jwtManager.GenerateKeyPair(finalKeyPath); err != nil {
 		logger.WithError(err).Error("Failed to generate keypair")
 		return err
 	}
-	
-	publicKey, err := os.ReadFile(publicKeyPath)
-	if err != nil {
+
+	if _, err := os.ReadFile(publicKeyPath); err != nil {
 		logger.WithError(err).Error("Failed to read generated public key")
 		return err
 	}
-	
+
 	fmt.Println("\n🔑 JWT Keypair Generated Successfully!")
 	fmt.Printf("📁 Location: %s\n", finalKeyPath)
 	fmt.Printf("🔒 Private Key: %s\n", privateKeyPath)
 	fmt.Printf("🔓 Public Key: %s\n", publicKeyPath)
-	fmt.Println("\n📋 Public Key for Registration:")
-	fmt.Println("=================================")
-	fmt.Print(string(publicKey))
+	fmt.Println("\n⚠️  IMPORTANT: Back up these keys! Losing them will require re-registration.")
 	fmt.Println("=================================")
 	fmt.Println("\n💡 Next Steps:")
-	fmt.Println("1. Register the public key above with your P0 backend")
-	fmt.Println("2. Keep the private key secure and backed up")
-	fmt.Printf("3. Run: p0-ssh-agent start --org-id YOUR_ORG --host-id YOUR_HOST --key-path %s\n", finalKeyPath)
```

Comment:
I removed step 3. because starting the p0-ssh-agent as a service is printed after successful registration.